pandas_sphinx_theme/layout.html

```diff
@@ -38,32 +38,42 @@
   <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="60">
 {%- endblock %}
 {%- block content %}
+    {% block docs_navbar %}
     <nav class="navbar navbar-light navbar-expand-lg bg-light fixed-top bd-navbar" id="navbar-main">
       {%- include "docs-navbar.html" %}
     </nav>
+    {% endblock %}
 
     <div class="container-fluid">
       <div class="row flex-xl-nowrap">
+          {% block docs_sidebar %}
           <div class="col-12 col-md-3 col-xl-2 bd-sidebar">
               {%- include "docs-sidebar.html" %}
           </div>
+          {% endblock %}
 
+          {% block docs_toc %}
           <div class="d-none d-xl-block col-xl-2 bd-toc">
               {% if not (meta is not none and 'notoc' in meta) %}
                 {%- include "docs-toc.html" %}
               {% endif %}
             </div>
+          {% endblock %}
 
+          {% block docs_main %}
```

Comment:
sphinx also has a "document" block already, that might be reused here

Reply:
The main reason I wasn't re-using blocks was just in case we were accidentally over-riding something else that was used. Do you think it's fine to use `document`?